composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "Sheer/knowlegebase",
+    "name": "Sheer/knowledgebase",
```

Comment:
@Galixte: English on GitHub :)

Renaming an extension need to update the namespace in PHP files. But this action are also side effects: To change the name, we need to **purge** the extension, resulting in loss of datas.